webpack.config.js

```diff
@@ -18,37 +18,45 @@ const { getIfUtils, removeEmpty } = require( 'webpack-config-utils' );
 
 
 module.exports = ( env ) => {
-  const { ifProd, ifNotProd, ifTest, ifNotTest } = getIfUtils( env );
+  const { ifProd, ifNotProd, ifTest, ifDev, ifSite } = getIfUtils( env, ['prod', 'test', 'dev', 'site'] );
+
+  function _packagePrefix() {
+    return './packages' + ( env.element ? `/${env.element}` : '' );
+  }
+
+  function _entryPoint() {
+    if ( ifTest() ) {
+
+      return {
+        'test': _packagePrefix() + '/index.test.ts',
+        'test-helpers' : './test-helpers.ts'
+      };
+
+    }
+
+    if ( ifProd() ) {
+
+      return {
+        'index': _packagePrefix() + '/index.ts',
```

Comment:
If we add **externals**, skate will not make it to any of the bundles, as externals are global, or at least I have not found anything else in Webpack "documentation".

As far as polyfills goes - I think we did not want to ship them at all.